rootston/seat.c

```diff
@@ -877,33 +877,33 @@ static void seat_add_pointer(struct roots_seat *seat,
 }
 
 static void handle_switch_destroy(struct wl_listener *listener, void *data) {
-	struct roots_switch *lid_switch =
-		wl_container_of(listener, lid_switch, device_destroy);
-	struct roots_seat *seat = lid_switch->seat;
+	struct roots_switch *switch_device =
+		wl_container_of(listener, switch_device, device_destroy);
+	struct roots_seat *seat = switch_device->seat;
 
-	wl_list_remove(&lid_switch->link);
-	wl_list_remove(&lid_switch->device_destroy.link);
-	free(lid_switch);
+	wl_list_remove(&switch_device->link);
+	wl_list_remove(&switch_device->device_destroy.link);
+	free(switch_device);
 
 	seat_update_capabilities(seat);
 }
 
 static void seat_add_switch(struct roots_seat *seat,
 		struct wlr_input_device *device) {
 	assert(device->type == WLR_INPUT_DEVICE_SWITCH);
-		struct roots_switch *lid_switch = calloc(1, sizeof(struct roots_switch));
-	if (!lid_switch) {
+		struct roots_switch *switch_device = calloc(1, sizeof(struct roots_switch));
```

Comment:
There is an extra indentation level here. Since the line is being modified anyway, can you please remove it?

Reply:
Fixed.